lib/src/reboot.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) fn reboot() -> anyhow::Result<()> {
     let _ = std::io::stdout().flush();
     let _ = std::io::stderr().flush();
     Task::new("Rebooting system", "systemd-run")
-        .args(["--message=Initiated by bootc", "reboot"])
+        .args(["--message=Initiated by bootc", "echo", "hello world"])
```

Comment:
To test, added a temporary change to intentionally break reboot.  
Here we're definitely not expecting the test to pass since there's no possible way for reboot to be called.

(I'm not expecting "hello world" to be outputted anywhere as I would assume the output is hidden inside the systemd service logs)